webpack.dev.config.babel.js

```diff
@@ -35,9 +36,7 @@ const assetsPath = path.resolve(__dirname, 'dist');
 
 const hmr = `webpack-hot-middleware/client?path=//${webpackHost}:${webpackPort}/__webpack_hmr`;
 
-const appName = config.get('appName');
-
-const entryPoints = { [appName]: [hmr, `${appName}/client`] };
+const entryPoints = { [APP_NAME]: [hmr, `${APP_NAME}/client`] };
```

Comment:
I think we should introduce a constant named `WEBPACK_ENTRYPOINT` ( set to the value of `APP_NAME`) and we could use that here and in the code-splitting code. It's important that we use the same value in the webpack config and in the code-splitting code.